tools/object_generator.go

```diff
@@ -125,6 +137,11 @@ import (
 	"go.anx.io/go-anxcloud/pkg/api/types"
 )
 `)
+	} else if gen.mode == "runtime" {
```

Comment:
the `if gen.mode == "docs"` above is a leftover from a idea I had when starting this code generator .. but that comment it adds is useful for all generated go code, so please add it for modes `tests` and `runtime`.